### PR TITLE
OAuth for Okta Service App FixtIt Day Clarity Update

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** OAuth for Okta Service App FixtIt Day Clarity Update
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-666914](https://oktainc.atlassian.net/browse/OKTA-666914)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md](https://github.com/okta/okta-developer-docs/blob/1230ad28825e45d7edba11bcb867db47d49c2825/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-oauth-for-okta-serviceapp%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/934ded9c-1bcb-421c-b85d-f05905d70380) | 100  | 76  | 100  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.bfb02613-6883-4656-a29b-b08f76695862?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md [hunk 1]
```diff
@@ -26,18 +28,24 @@ This guide explains how to interact with Okta APIs by using scoped OAuth 2.0 acc
 
 Most Okta API endpoints require that you include an API token with your request. Currently, this API token takes the form of an SSWS token that you generate in the Admin Console. With OAuth for Okta, you’re able to interact with Okta APIs using scoped OAuth 2.0 access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
 
+Scoped access tokens have several advantages, including:
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md:31]:
```suggestion
Scoped access tokens have several advantages:
```

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md [hunk 2]
```diff
@@ -113,13 +123,13 @@ The private key that you use to sign the JWT must have the corresponding public
     * Key size: 2048
     * Key use: signature
     * Algorithm: RSA256
-    * Key ID: This can be any random value.
+    * Key ID: This can be any random value
```
Comment (grahamsmith-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md:126]: for consistency with the previous bullets, I think you could just make this

any random value

(that way it's lowercase like "signature")...but up to you. I'm fine either way

**Review by grahamsmith-okta — APPROVED** [review]

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md](https://github.com/okta/okta-developer-docs/blob/6c7d53ac42b480ddd6d2767dfc032f62fd16eea1/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-oauth-for-okta-serviceapp%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/0a363de7-980a-484a-bf03-9c64d800c12f) | 100  | 76  | 100  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.1b9e5cf2-c12e-4335-93ed-f82528d87c13?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md (hunk 2 above)
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md:126]:
```suggestion
    * Key ID: any random value
```

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/main/index.md](https://github.com/okta/okta-developer-docs/blob/b897e714d6844fc8b62959ddae338604c1ed2b1d/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fimplement-oauth-for-okta-serviceapp%2Fmain%2Findex.md) | [88](https://okta.acrolinx.cloud/api/v1/checking/scorecards/6fe33534-e1fc-4e38-abee-98fb0fd56398) | 100  | 76  | 100  | 97  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b9c6a9bf-6252-4344-8259-6aa8f3a62528?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by grahamsmith-okta — APPROVED** [review]

**Pull request merged**